frontend/src/components/PixelInspector.tsx

```
@@ -15,28 +21,39 @@ interface DeckHoverInfo {
+  const px = Math.min(
+    width - 1,
+    Math.max(0, Math.floor(((lng - west) / spanX) * width))
+  );
+  const py = Math.min(
+    height - 1,
+    Math.max(0, Math.floor(((north - lat) / spanY) * height))
+  );
 
-  const val = entry.data[py * entry.width + px];
+  const val = raw[py * width + px];
   if (val !== val) return null;
```

Biome (2.4.11)

[error] Comparing to itself is potentially pointless.

(lint/suspicious/noSelfCompare)